### ISSUE TITLE
Monk Demolish and Twin Snakes refresh time is wrong

### ISSUE THREAD

**Issue by hoskid** [MKhayle/XIVComboPlugin#228]
Demolish and Twin Snakes should be refreshed when they have less than 5 seconds not less than 6 seconds.

Here is a cheat sheet from The Balance Discord https://i.imgur.com/C5lQhpe.png

**Comment by kaedys** [comment]
Bit odd that it's 5s, given that the Mock default GCD is in the neighborhood of 2s, and the common builds typically run just under 2s.  If the buff was at 5.1s (or anything under GCDx3 plus some wiggle room for latency) and you used True Strike, by the time you got back around to Twin Snakes, the buff would have fallen off.  True Strike only has a 20p gain over Twin Snakes (~7.1%), while the Twin Snakes buff provides 15% total damage increase.  Twin Snakes without the buff is a loss of 42 potency versus with the buff, so using True Strike and letting the buff fall off for the next Twin Snakes loses you more than you gain.  You'd need a GCD of not more than 1.66s for a 5s threshold to still capture the next Twin Snakes inside the buff.

In fact, their ["basic guide" to monks](https://www.thebalanceffxiv.com/jobs/melee/monk/basic-guide/) cites 7s as the threshold instead.  The Icy-Veins guide, which was written by *Perfect Balance*, the same person who wrote all of the pinned posts in the `mnk_resources` channel on the Balance Discord, cites 6s.  

Basically, it's fine, imo.  In fact, 5s is probably technically wrong, and I'd wager it's in that image more because they thought it'd be easier to remember, and because if you're doing the rotation _correctly_ (in a vacuum with no downtime, of course), it doesn't actually matter, because you simply alternate between True Strike and Twin Snakes.

**Comment by MKhayle** [comment]
#206 led to changes which propose a decent alternative for now, so I'm closing this one
